test_serve.py

```diff
@@ -76,9 +76,8 @@ def test_server_info():
         "cli-minimal-version": "0.14.1",
         "cli-bad-versions": [],
         "services": {
-            "girder": {"url": "https://girder.dandiarchive.org"},
+            "api": {"url": "https://api.dandiarchive.org/api"},
```

Comment:
Let's keep it available if only for just some time to avoid blunt KeyError (`dandi-cli` used to do that)
```suggestion
            "girder": {"url": None},
            "api": {"url": "https://api.dandiarchive.org/api"},
```